tests/ballerina-integration-test/src/test/resources/socket/services/02_server_socket_join_departure.bal

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/socket;
+
+listener socket:Listener server = new ({
+    port:61598
```

Comment:
Do we need to stick with the convention we use for i.e http ? where port will be taken in separately as a constructor argument and configurations as records ?

Reply:
+1. I will incorporate that changes with the next iteration.